resources/boards/OAK-D-PRO-W.json

```diff
@@ -5,7 +5,7 @@
         "revision": "R3M1E3",
         "swap_left_and_right_cameras": true,
         "left_fov_deg": 127.0,
-        "rgb_fov_deg": 127.0,
+        "rgb_fov_deg": 108.0,
```

Comment:
Querying fov from intrinsic vs specs will be significantly different.
I think this is expected, and intended. Just mentioning.

Reply:
True. But also good to have Proper full FOv value there in case needed for future.